src/test/transaction.mock.ts

```diff
@@ -0,0 +1,10 @@
+import { TransactionFactory, type TxData } from '@ethereumjs/tx';
+
+export const baseTransactionMockFactory = (options: TxData = {}) =>
```

Comment:
Nit: While this name is accurate — it is indeed a factory — it seems a bit unusual the name of this variable is a noun and not a verb phrase. What do you think about `buildMockTransaction`? Then again, if this is intended to mirror `keyringBuilderFactory`, then that's fine.

Reply:
Indeed I initially named it after the `keyringBuilderFactory`, but what you are saying makes more sense - `buildMockTransaction` sounds good to me, refactored in 988974e